src/components/ViolationForm.js

```diff
@@ -0,0 +1,250 @@
+import { useForm } from 'react-hook-form';
+import styled from 'styled-components';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+
+const CommentFormStyle = styled.form`
+  width: 100%;
+
+  .errorMsg {
+    color: red;
+  }
+  .radioLabel {
+    float: left;
+    clear: none;
+    display: block;
+    padding: 0px 1em 0px 8px;
+  }
+  input[type='radio'] {
+    float: left;
+    clear: none;
+    margin: 2px 0 0 2px;
+  }
+  label {
+    display: block;
+    margin-left: 5px;
+  }
+  input {
+    width: 80%;
+    font-size: 18px;
+    padding: 20px;
+    border: 1px solid #eee;
+    margin: 20px 0;
+    box-sizing: border-box;
+    margin-left: 5px;
+  }
+`;
+
+export default function ViolationForm() {
```

Comment:
Let's embed this form under a route for now.

Reply:
What do you mean exactly?